examples/tic-tac-toe/server/server.bal

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/io;
+import ballerina/websocket;
+
+const string USER_SIGN = "userSign";
+const string SIGN_X = "X";
+const string SIGN_0 = "0";
+map<websocket:Caller> connectionsMap = {};
+string[9] squares = [];
+boolean started = false;
+string next = "";
+string winner = "";
+type Message record {
+    string 'type;
+    int move;
+    string next;
+    string player;
+};
+
+service /ws on new websocket:Listener(8000) {
+    resource function get game() returns websocket:Service|websocket:UpgradeError {
+        // The server can accept a WebSocket connection by returning a `websocket:Service`.
+        return new GameServer();
+    }
+}
+
+service class GameServer {
+    *websocket:Service;
+
+    remote function onOpen(websocket:Caller caller) returns error? {
+        json welcomeMsg;
```

Comment:
No, it can go into the else block. Fixed with [245d2fd](https://github.com/ballerina-platform/module-ballerina-websocket/pull/655/commits/245d2fd99f3a995f0e2ab8c1686f478ca5136334)